container/shim/src/index.js

```diff
@@ -91,9 +95,10 @@ if (cluster.isPrimary) {
     })
 
     if (req.headers.range) {
-      let [start, end] = req.headers.range.split('=')[1].split('-')
-      start = parseInt(start, 10)
-      end = parseInt(end, 10)
+      const range = req.headers.range.split('=')?[1]?.split('-')
+      assert(range, 'TODO')
```

Comment:
```suggestion
      if(!range) {
        return res.status(400).send('Invalid range')
      }
```

Reply:
This is where the error handling comes in. I actually meant to use https://www.npmjs.com/package/http-assert:

```suggestion
      httpAssert(range, 400, 'invalid range')
```

If the async route handler now forwards this to `next`, then the express final middleware can take care of responding with the appropriate error per request

Reply:
Can you add the update it with it?